index.html

```diff
@@ -126,8 +126,12 @@ <h3 class="work-role h3-common-styles">Back End Dev</h3>
                   <button type="button" class="works-btn">javascript</button>
                 </li>
               </ul>
-              <button class="see-project-btn" type="button">See Project</button>
+              <button class="see-project-btn seeWork1Btn" type="button">
+                See Project
+              </button>
             </div>
+            <!-- ---------------Popup Window----------------- -->
+            <section class="popupWindow hide"></section>
           </div>
         </div>
 
```

Comment:
- [ ] While you did a good job creating dynamically the popup :+1:, I think it is one of the main requirements to create the original cards that you previously created in HTML  by using javascript so kindly remove all the code for all the cards from the HTML (the projects/works section) and create it by using JS. It would help you to learn how to create HTML by JS :100:. [Here](https://github.com/olawale-o/portfolio-mobile-version/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95) you can find a good example of it.